pkg/ebpf/tracer.go

```diff
@@ -438,8 +438,13 @@ func (t *Tracer) removeEntries(mp, tcpMp *bpflib.Map, entries []*ConnTuple) {
 	for i := range entries {
 		err := t.m.DeleteElement(mp, unsafe.Pointer(entries[i]))
```

Comment:
would DeleteMap return an error under any other circumstance?

Reply:
[Last time](https://github.com/iovisor/gobpf/pull/186), I looked at this, it [returns an error[(https://github.com/iovisor/gobpf/blob/master/elf/table.go#L130-L137) if the syscall fails or if the element is not found

Reply:
In either case I think it's safe to keep the code as it is.
If it's a system call failure, which isn't exactly what we're expecting, we'll re-attempt to delete the entry in the next `GetActiveConnections` call.